src/apm.coffee

```diff
@@ -61,7 +61,7 @@ module.exports =
     process.env.ATOM_API_URL ? 'https://atom.io/api'
 
   getNodeVersion: ->
-    process.env.ATOM_NODE_VERSION ? '0.22.0'
+    process.env.ATOM_NODE_VERSION ? require(path.join(@getResourcePath(), 'package.json'))?.electronVersion
```

Comment:
`@getResourcePath` is async so I don't think you'd be able to call it like this.